articles/azure-app-configuration/howto-leverage-json-content-type.md

```diff
@@ -168,7 +168,7 @@ az appconfig kv export -d file --format json --path "~/Export.json" --separator
 ```
 
 > [!NOTE]
-> If your App Configuration store has some key-values without JSON content-type, they will also be exported to the same file in string format. If you want to export only the JSON key-values, assign a unique label or prefix to your JSON key-values and use label or prefix filtering during export.
+> If your App Configuration store has some key-values without JSON content-type, they will also be exported to the same file in string format.
```

Comment:
I agree that we shouldn't suggest assigning label just for exporting purposes. But there might be some users who just want to export their JSON key-values. Unless we offer content type filtering, label/prefix filtering can be leveraged to get the desired output. How about we change the wording such that we don't directly suggest assigning a label for export but implicitly hint that there is a way to filter by label/prefix in case users really need it:

`If your App Configuration store has some key-values without JSON content-type, they will also be exported to the same file in string format. Filtering by content-type is not supported. You can only filter key-values by label or prefix during export.`

Reply:
Documenting unsupported feature isn't appealing to begin with. But most importantly, filtering by content-type won't solve the issue. Media queries have specific parsing rules. For example application/json, application/vnd.customtype+json;charset="utf-8", etc., all are valid json media, but quite different from filtering point of view.
I think the solution sides on client. Request what's needed (or perhaps all) and then export only these kvs that match the media query content-type.

Reply:
Makes sense, thanks!